en/docs/deploy/security/configure-keystores-in-wso2-products.md

```diff
@@ -19,20 +19,20 @@ The `wso2carbon.jks` keystore file, which is shipped with WSO2 Identity Server,
 
--   **Encrypting/decrypting** passwords and other confidential information, which are maintained in various configuration files as well as internal data stores. Note that you also have the option of
-    [separating the keystore for encrypting information in internal data stores](#configure-a-separate-keystore-for-encrypting-data-in-internal-data-stores).
+-   **Encrypting/decrypting** passwords and other confidential information, which are maintained in various configuration files as well as internal datastores. Note that you also have the option of
+    [separating the keystore for encrypting information in internal datastores](#configure-a-separate-keystore-for-encrypting-data-in-internal-data-stores).
 -   **Signing messages** when WSO2 Identity Server communicates with external parties (such SAML, OIDC id_token signing). 
 
 ---
 
-## Configure a separate keystore for encrypting data in internal data stores
+## Configure a separate keystore for encrypting data in internal datastores
 
 !!! info 
-    This is available as part of the newly introduced Crypto Service. It is an extensible framework that facilitates the cryptography needs of WSO2 Identity Server.
+    This is available as part of the Crypto Service. It is an extensible framework that facilitates the cryptography needs of WSO2 Identity Server.
 
-Currently, the primary keystore configured in `deployment.toml` is used for internal data encryption (encrypting data in internal data stores and configuration files) as well as for signing messages that are communicated with external parties. However, it is sometimes a common requirement to have separate keystores for communicating messages with external parties (such SAML, OIDC id_token signing) and for encrypting information in internal data stores. This is because, for the first scenario of signing messages, the keystore certificates need to be frequently renewed. However, for encrypting information in internal data stores, the keystore certificates should not be changed frequently because the data that is already encrypted will become unusable every time the certificate changes.
+Currently, the primary keystore configured in `deployment.toml` is used for internal data encryption (encrypting data in internal datastores and configuration files) as well as for signing messages that are communicated with external parties. However, it is sometimes a common requirement to have separate keystores for communicating messages with external parties (such SAML, OIDC id_token signing) and for encrypting information in internal datastores. This is because, for the first scenario of signing messages, the keystore certificates need to be frequently renewed. However, for encrypting information in internal datastores, the keystore certificates should not be changed frequently because the data that is already encrypted will become unusable every time the certificate changes.
```

Comment:
`deployment.toml` -> shouldn't we provide the directory path for this file?

Reply:
We have given it in the steps below.